src/azure-cli/azure/cli/command_modules/role/custom.py

```diff
@@ -20,7 +20,6 @@
 import uuid
 
 import dateutil.parser
-from dateutil.relativedelta import relativedelta
```

Comment:
Although `dateutil.relativedelta.relativedelta` also supports `days`, `dateutil` is third-party:

https://docs.python.org/3/library/datetime.html

> Package [dateutil](https://dateutil.readthedocs.io/en/stable/)
> Third-party library with expanded time zone and parsing support.

Third-party library dependencies should be as less as possible.